src/data.convert.js

```diff
@@ -131,16 +130,25 @@ c3_chart_internal_fn.convertColumnsToData = (columns) => {
             }
             newRows[j - 1][key] = columns[i][j];
         }
+        keys.push(key);
     }
 
-    return newRows;
+    return { keys, rows: newRows };
 };
 
 c3_chart_internal_fn.convertDataToTargets = function (data, appendXs) {
```

Comment:
Now the shape of `data` is changed to `{ keys, rows }`. I think we should document that in jsdoc format like:

```
/**
 * Converts the data format into the target format.
 * @param {Array} data.keys ...describe...
 * @param {Array} data.rows ...describe...
 * @param {boolean} appendXs
 */
```